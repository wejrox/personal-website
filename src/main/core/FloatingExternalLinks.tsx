import * as React from "react";
import { GoMarkGithub } from "react-icons/go";
import { FaLinkedin } from "react-icons/fa";
import IconLink from "../components/IconLink";
import Col from "reactstrap/lib/Col";

/**
 * FloatingExternalLinks located at the bottom right of the page, fixed.
 */
class FloatingExternalLinks extends React.Component {
    public render() {
        return (
            <div className={"floating-links"}>
                <Col className={"d-flex justify-content-end"}>
                    <IconLink link={"https://www.github.com/wejrox/"} icon={<GoMarkGithub size={32}/>}/>
                    <IconLink link={"https://www.linkedin.com/in/james-mcdowell001/"} icon={<FaLinkedin size={32}/>}/>
                </Col>
            </div>
        );
    }
}

export default FloatingExternalLinks;
