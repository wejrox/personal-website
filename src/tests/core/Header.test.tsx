import * as Enzyme from "enzyme";
import * as React from "react";
import FloatingExternalLinks from "../../main/core/FloatingExternalLinks";

describe("FloatingExternalLinks Component", () => {
    const wrapper = Enzyme.shallow(<FloatingExternalLinks/>);

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
