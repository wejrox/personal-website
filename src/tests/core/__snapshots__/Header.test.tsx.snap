// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatingExternalLinks Component renders correctly 1`] = `ShallowWrapper {}`;
